stdlib/int32.mli

```diff
@@ -52,12 +52,24 @@ external div : int32 -> int32 -> int32 = "%int32_div"
    argument is zero.  This division rounds the real quotient of
    its arguments towards zero, as specified for {!Pervasives.(/)}. *)
 
+val div_unsigned : int32 -> int32 -> int32
+(** Same as [div], except that arguments and result are interpreted as {e
```

Comment:
Using `{!div}` here rather than `[div]` would provide the reader with a direct link. (The comment applies for more docstrings below).

Reply:
Thanks, fixed.